src/doc/components/Esdoc/SummaryDoc.js

```diff
@@ -46,9 +47,12 @@ const SummaryDoc = ({
           if (['get', 'set'].includes(doc.kind)) {
             kind = doc.kind;
           }
+          */
```

Comment:
No need to comment this code

Reply:
I remove them then since they are unused.

Reply:
sorry i didn't want  you remove it . I wanted it uncomment it could be used in the future.Particularly the static part